spec/support/messaging/cy.yml

```diff
@@ -0,0 +1,313 @@
+et3-v1-cy:
+  response_pdf_fields:
+    header:
+      case_number:
+        field_name: 'Rhif yr'
+      date_received:
+        field_name: false
+      rtf:
+        field_name: false
+    claimant:
+      claimants_name:
+        field_name: 'undefined'
+    respondent:
+      name:
+        field_name: 'undefined_2'
+      contact:
+        field_name: 'undefined_3'
+      address:
+        building:
+          field_name: 'Rhif neu enw'
+        street:
+          field_name: 'Stryd'
+        locality:
+          field_name: 'TrefDinas'
+        county:
+          field_name: 'Sir'
+        post_code:
+          field_name: 'Cod post'
+      address_dx_number:
+        field_name: 'undefined_4'
+      phone_number:
+        field_name: 'Lle gallwn gysylltu â chi yn ystod y dydd'
+      mobile_number:
+        field_name: 'n wahanol'
+      contact_preference:
+        field_name: false
+      email_address:
+        field_name: 'undefined_5'
+      fax_number:
+        field_name: 'undefined_6'
+      employ_gb:
+        field_name: 'sefydliad hwn ym Mhrydain Fawr'
+      multi_site_gb:
```

Comment:
This is a new way of representing a field - you will notice it has 2 field names.  This is when you have multiple radio buttons all named differently, that we want to bring together into 1 value.  So, the 'select_values' and 'unselected_value' list the values that each field should be set to in order to represent the value specified.